azure-function-http/src/test/groovy/io/micronaut/azure/function/http/ParameterBindingSpec.groovy

```diff
@@ -179,6 +180,7 @@ class ParameterBindingSpec extends Specification {
         function.close()
     }
 
+    @PendingFeature
```

Comment:
i think this one is https://github.com/micronaut-projects/micronaut-servlet/pull/837/files#diff-58cdcb7da423882faa47ed935acf6ab43b1b1b66f86109c32e18b1f1377805fcR119

basically `@Header` on controllers is not a thing in core